examples/modules/01-nested-modules/example.effekt

```diff
@@ -0,0 +1,15 @@
+module hello {
+    def greet(name: String) = {
+        println(concat("Hello ", name))
+    }
+
+    module world {
+        def greet() = {
+            greet("World")
```

Comment:
Design Decision: Should nested modules implicitly import their parent?

Reply:
I would go with yes, it is just like nested scopes.